scripts/tests/py/test_metadata.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+from os import path
+
+from metadata import Metadata, MetadataReader
+
+
+class TestMetadataReader(unittest.TestCase):
+    path_under_test = path_under_test = path.join(path.dirname(__file__), "simple_run_args.txt")
+
+    def setUp(self):
+
+        # build the reader object
+        self.reader = MetadataReader(path.join(path.dirname(__file__), "env_test.yaml"))
+        with open(self.path_under_test, 'w', encoding='utf8') as test_file:
+            test_file.writelines(
+                ["# test-runner-runs: run1", "\n# test-runner-run/run1: app/all-clusters discriminator KVS storage-path commissioning-method discriminator passcode"])
```

Comment:
This is hard to read. Use multiline strings

test_file.write('''
# test-runner-runs: run1
# test-runner-run/run1: app/all-clusters discriminator KVS storage-path commissioning-method discriminator passcode

non-parsed string
''')